tests/integration_test/ss2_notification_dss_staging.json

```diff
@@ -4,6 +4,9 @@
   "match": {
     "bundle_uuid": "d0686b4d-a4b7-45e2-89b7-ac0ebdb4a6b7",
     "bundle_version": "2018-01-08T221736.018244Z"
+  },
+  "labels": {
+    "integration-test-workflow": "true"
```

Comment:
how about "mintegration-test"
workflow is implied and this can also help us distinguish these workflows from those triggered by dcp wide testing